khmer/_oxli/graphs.pyx

```diff
@@ -439,7 +439,7 @@ cdef class Hashgraph(Hashtable):
     def neighbors(self, object kmer):
         '''Get a list of neighbor nodes for this k-mer.'''
         cdef Traverser traverser = Traverser(self)
-        return [str(n) for n in traverser._neighbors(self._build_kmer(kmer))]
+        return list(traverser._neighbors(self._build_kmer(kmer)))
```

Comment:
I've been trying to push patterns like this toward iterator interfaces with the Cython stuff. Not a deal breaker, but something to consider.

Reply:
hey maaan I'm just riffing off your code :).

yes, on board with that.